### PR TITLE
Update to latest release for npm packages

### LINKED ISSUE #944: Fix unmet dependencies and update npm packages
Updating packages to latest release and resolving some unmet dependencies. 

- electron: 1.7.9

- patternfly : 3.29.5

- eslint : 4.9.0

### PR DESCRIPTION
**Pull request by mohitsuman**
Fixes #944

### CODE REVIEW

**Comment by dgolovin** [comment]
test this

File: package.json [hunk 1]
```diff
@@ -79,29 +78,29 @@
     "gulp-rename": "1.2.2",
     "gulp-run": "1.7.1",
     "gulp-sourcemaps": "^2.4.1",
-    "gulp-spawn-mocha": "3.3.1",
+    "gulp-spawn-mocha": "4.0.1",
     "gulp-symlink": "2.1.4",
     "gulp-unzip": "0.2.0",
     "gulp-zip": "4.0.0",
-    "istanbul": "1.1.0-alpha.1",
+    "istanbul": "0.4.5",
```
Comment (dgolovin) [inline package.json]: Downgrading istanbul leads to 
``` shell
No coverage information was collected, exit without writing coverage information
```

Comment (mohitsuman) [reply package.json]: I had updated instanbul with the stable release, but yes the coverage was failing. Reverted back to `"istanbul": "1.1.0-alpha.1"`.

**Comment by mohitsuman** [comment]
test this

**Comment by mohitsuman** [comment]
test this

**Pull request merged**